sorc/build_ufs_coupled.sh

```diff
@@ -13,15 +13,16 @@ if [ $target = hera ]; then target=hera.intel ; fi
 if [ $target = orion ]; then target=orion.intel ; fi
 if [ $target = stampede ]; then target=stampede.intel ; fi
 
-MOD_PATH=$cwd/ufs_coupled.fd/modulefiles/$target
+MOD_PATH=$cwd/ufs_coupled.fd/modulefiles
 
 module purge 
 module use $MOD_PATH 
-module load fv3 
+module load ufs_${target}
 cd ufs_coupled.fd/
 if [[ -d build ]]; then rm -Rf build; fi
 if [[ -d GOCART ]]; then
```

Comment:
So you have switched to using your fork, but as soon as this PR goes into ufs-weather-model this build check will not work anymore.  Do we address that now or later?

Reply:
In the most recent versions of `ufs-weather-model`all modulefiles have been renamed as: `ufs_<machine>.<compiler_settings>`. This change introduces the update and should work from now on.

Reply:
But you didn't update the hash of what is checked out for the ufs-weather-model for the coupled case.  Therefore this will break the coupled case as is,  unless I missed that.

Reply:
This was my issue as well. The currently non-aerosol coupled-crow ufs model version is still using the old naming convention. So this PR would break that until #324 is also merged.

Reply:
(And vice versa)

Reply:
Well if the changes were only within the if/else statements it could be fine, but as it is this will break atm-ocn-atm-wav

Reply:
You are right. My assumption was that we were going to update to the latest ufs-weather-model for the coupled case, too, before or at the same time as this PR. I will address this issue by adding a conditional block.

Reply:
I hope we will update ufs-weather-model for the other case very soon, but as of now #324 is remains an unassigned task.

Reply:
Commit f81741bdb3a8cc221f6e0846c706cfe3cc9c4010 addresses this issue. Updated modulefiles are only loaded to build the ufs-weather-model with prognostic aerosols. This model is identified though a hidden file, created when using option `-a` during checkout.

Reply:
Looks like I'm too late and the complications have already been added, but I'm going to try and tackle #324 after we clear a couple of the other PRs out. So let's cool it with adding more if-block exceptions for aerosols when we want to be removing them.

Reply:
At least until we can discuss it in the afternoon.